ddocs/medic-db/medic/views/doc_summaries_by_id/map.js

```diff
@@ -33,6 +34,8 @@ function(doc) {
     var subject = {};
     var reference = doc.patient_id ||
                     (doc.fields && doc.fields.patient_id) ||
+                    doc.patient_uuid ||
```

Comment:
`doc.patient_uuid` is not a property that is used by the CHT. `patient_uuid` should only be part of `fields`. If we add this to the view, we would validate the support for `doc.patient_uuid`, while we do not actually support it.